_source/logzio_collections/_prometheus-sources/go-custom-metrics.md

```diff
@@ -34,196 +32,238 @@ Go 1.x or higher
-	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
-	"go.opentelemetry.io/otel/sdk/resource"
-	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
-)
+Replace the placeholders in the code to match your specifics.
 
-func main() {
-	// Create a new Config struct.
-	config := cortex.Config{
-		Endpoint:      "<<LISTENER-HOST>>",
-		RemoteTimeout: 30 * time.Second,
-		PushInterval:  15 * time.Second,
-		BearerToken:   "<<PROMETHEUS-METRICS-SHIPPING-TOKEN>>",
-	}
-	// Create and install the exporter. Additionally, set the push interval to 15 seconds
-	// and add a resource to the controller.
-	pusher, err := cortex.InstallNewPipeline(config,
-		controller.WithCollectPeriod(15*time.Second),
-		controller.WithResource(resource.NewWithAttributes(
-			semconv.SchemaURL,
-			attribute.String("p8s_logzio_name", "LABEL_NAME")),
-		),
-	)
-	if err != nil {
-		fmt.Printf("Error: %v", err)
-	}
-	ctx := context.Background()
-	defer handleErr(pusher.Stop(ctx))
-	// Create a counter and a value recorder
-	meter := pusher.MeterProvider().Meter("go_metrics")
-	// Create instruments
-	counter := metric.Must(meter).NewInt64Counter(
-		"go_metrics.counter",
-		metric.WithDescription("Counts things"),
-	)
-	// Start the metrics pipline
-	pusher.Start(ctx)
-	// Record values to the instruments and add labels
-	counter.Add(ctx, 10, attribute.String("key", "value"))
-}
-func handleErr(err error) {
-	if err != nil {
-		fmt.Println("Encountered error: ", err.Error())
-	}
+| Parameter | Description | Required | Default|
+|---|---|---|---|
+|`<<LISTENER-HOST>>`|  The full Logz.io Listener URL for for your region, configured to use port **8052** for http traffic, or port **8053** for https traffic (example: https://listener.logz.io:8053). For more details, see the [regions page](https://docs.logz.io/user-guide/accounts/account-region.html) in logz.io docs | Required | - |
+|`<<PROMETHEUS-METRICS-SHIPPING-TOKEN>>`| The Logz.io Prometheus Metrics account token. Find it under **Settings > Manage accounts**. [Look up your Metrics account token.](https://docs.logz.io/user-guide/accounts/finding-your-metrics-account-token/)  | Required | - |
+| RemoteTimeout | The timeout for requests to the remote write Logz.io metrics listener endpoint. | Required | 30 (seconds) |
+| PushInterval | The time interval for sending the metrics to Logz.io. | Required | 10 (seconds) |
+| Quantiles | The quantiles of the histograms. | Optional | [0.5, 0.9, 0.95, 0.99] |
+| HistogramBoundaries | The histogram boundaries. | Optional | - |
+
+##### Add the exporter setup
+
+Add the exporter setup definition to your application code:
+
+```go
+// Use the `config` instance from last step.
+
+cont, err := metricsExporter.InstallNewPipeline(
+    config,
+    controller.WithCollectPeriod(<<COLLECT_PERIOD>>*time.Second),
+    controller.WithResource(
+        resource.NewWithAttributes(
+            semconv.SchemaURL,
+            attribute.String("LABEL_KEY", "LABEL_VALUE"),
```

Comment:
Can you please replace this line with `attribute.<<TYPE>>("<<LABEL_KEY>>", "<<LABEL_VALUE>>"),`